updatecli/updatecli.d/charts/mirrorbits.yaml

```
@@ -0,0 +1,42 @@
+---
+name: Bump `mirrorbits-cli` version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  lastChartVersion:
+    kind: helmchart
+    name: get last chart version
+    spec:
+      url: https://jenkins-infra.github.io/helm-charts
+      name: mirrorbits
+
+targets:
+  updateChartVersion:
+    name: "Update the chart version for mirrorbits"
```

```suggestion
    name: Update the chart version for mirrorbits
```

(nit) no need for double quotes